src/Components/Admin/EnvironmentEditor/environment.js

```diff
@@ -57,6 +57,8 @@ export default function EnvironmentEditor({
     const [environment, setEnvironment] = useState({});
     const [scenes, setScenes] = useState([]);
     const [createModalOpen, setCreateModalOpen] = useState(false);
+    const [editModalOpen, setEditModalOpen] = useState(false);
+    const [editSceneCard, setEditSceneCard] = useState({});
```

Comment:
@aaronabraham311 @ahmedhamodi what do you guys think of this state name? I tried to name it something other than "editScene" to avoid a conflict with the "editScene" function in sceneEndpoints.js, but my naming sense is still pretty unrefined

Reply:
I think it's fine!

Reply:
Perhaps `editSceneInfo` would be better?